run.js

```diff
@@ -11,68 +12,120 @@ const pathFor = mode => path.join(
   'react-dom.' + mode + '.js'
 );
 
+// set up AST files.
 const ReactDOMProd = pathFor('production.min');
 const ReactDOMDev = pathFor('development');
 
-console.time('parse prod');
 const prod = fs.readFileSync(ReactDOMProd, 'utf8');
 const prodAst = babylon.parse(prod);
-console.timeEnd('parse prod');
+fs.writeFileSync('react-dom.production.min.ast', 'module.exports = ' + JSON.stringify(prodAst) + ';');
+fs.writeFileSync('react-dom.production.min.js', 'module.exports = ' + JSON.stringify(prodAst) + ';');
+fs.writeFileSync('react-dom.production.min.json', JSON.stringify(prodAst));
```

Comment:
do we want to the stringify-ing once since it could be a bit expensive (100s of ms, iirc).